pennylane/tape/qnode.py

```diff
@@ -40,11 +42,20 @@ class QNode:
 
     .. note::
 
-        As the quantum tape is a *beta* feature, the standard PennyLane
-        measurement functions cannot be used. You will need to instead
-        import modified measurement functions within the quantum tape:
+        The quantum tape is an *experimental* feature. QNodes that use the quantum
+        tape have access to advanced features, such as in-QNode classical processing,
+        but do not yet have feature parity with the standard PennyLane QNode.
+
+        This quantum tape-comaptible QNode can either be created directly,
+
+        >>> import pennylane.tape
+        >>> qml.tape.QNode(qfunc, dev)
+
+        or enabled globally via :func:`~.enable_tape` without changing your PennyLane code:
 
-        >>> from pennylane.beta.queuing import expval, var, sample, probs
+        >>> qml.enable_tape()
+
+        For more details, see :mod:`pennylane.tape`.

```

Comment:
@trbromley these links work for me?
![image](https://user-images.githubusercontent.com/2959003/94331547-d85df100-ffff-11ea-9015-587236e60759.png)

Or is there another reason you are suggesting for linking to the docs?

Reply:
Do we have a convention for :mod: vs :doc:?

Reply:
not really, this could also be replaced with `:doc:`. The only difference is how it is typeset; with `:mod:` it will be typeset using as code, whereas `:doc:` will be typeset as standard text.

I don't have a preference either way, so happy to change it!